### ISSUE TITLE
Failed to activate the atom-powerline package

### ISSUE THREAD

**Issue by georgefahmy** [quarterto/powerline#3]
[Enter steps to reproduce:]

1. install atom-powerline

**Atom**: 1.44.0 x64
**Electron**: 4.2.7
**OS**: Mac OS X 10.15
**Thrown From**: [atom-powerline](https://github.com/quarterto/powerline) package 1.0.0

### Stack Trace

Failed to activate the atom-powerline package

```
At this.properties.iconClassForPath is not a function

TypeError: this.properties.iconClassForPath is not a function
    at FileIcon.render (/packages/atom-powerline/lib/items/file-icon.js:25:32)
    at Object.initialize (/packages/terminal-tab/node_modules/etch/lib/component-helpers.js:42:31)
    at /packages/atom-powerline/lib/items/file-icon.js:20:8)
    at /packages/atom-powerline/lib/atom-tab-status-items-view.js:20:19
    at Array.flatMap (<anonymous>)
    at AtomTabStatusItemsView.render (/packages/atom-powerline/lib/atom-tab-status-items-view.js:18:5)
    at /packages/atom-powerline/lib/atom-tab-status-items-view.js:12:8)
    at /packages/atom-powerline/lib/atom-tab-status-items.js:32:18
    at PaneContainer.observePaneItems (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:443893)
    at WorkspaceCenter.observePaneItems (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:45276)
    at c.getPaneContainers.map.t (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:428450)
    at Array.map (<anonymous>)
    at Workspace.observePaneItems (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:428441)
    at Object.load (/packages/atom-powerline/lib/atom-tab-status-items.js:27:19)
    at Object.activate (/packages/atom-powerline/lib/atom-tab-status-items.js:14:9)
    at Package.activateNow (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3214834)
    at measure (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3214246)
    at Package.measure (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3211856)
    at activationPromise.activationPromise.Promise (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3214106)
    at new Promise (<anonymous>)
    at Package.activate (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3214049)
    at PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:382452)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:941081
    at r (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:935687)
    at n (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1124792)
    at t.bufferStream (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1124946)
    at Socket.e.on (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1123548)
    at Socket.emit (events.js:187:15)
    at Pipe._handle.close (net.js:606:12)
```

### Commands

```
     -5:44.9.0 terminal:paste (textarea.xterm-helper-textarea)
     -3:13.4.0 settings-view:open (textarea.xterm-helper-textarea)
  6x -2:30.1.0 core:backspace (input.hidden-input)
  2x -0:39.4.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
atom-powerline 1.0.0 
atom-python-virtualenv 1.0.4 
autocomplete-python 1.16.0 
dbclick-tree-view 1.6.3 
file-header 1.13.9 
python-black 0.7.0 
script 3.25.0 
tasks 2.10.1 
terminal-tab 0.6.0 
```

**Comment by apaleslimghost** [comment]
thanks for the issue, @georgefahmy! as a workaround, you can install the `file-icons` package which provides this service. i'll need a default for this [like `tree-view` does](https://github.com/atom/tree-view/blob/master/lib/default-file-icons.coffee)

**Comment by apaleslimghost** [comment]
thanks again @georgefahmy. this should be fixed in v1.1
